bokeh/models/tools.py

```diff
@@ -392,6 +392,41 @@ class BoxZoomTool(Drag):
 
     """)
 
+class ZoomInTool(Action):
+    """ *toolbar icon*: |zoom_in_icon|
+
+    The zoom-in tool allows users to click a button to zoom in
+    by a fixed amount.
+
+    .. |zoom_in_icon| image:: /_images/icons/ZoomIn.png
+        :height: 18pt
+
+    """
+    # TODO ZoomInTool dimensions should probably be constrained to be the same as ZoomOutTool
```

Comment:
Currently this is no tooling in bokeh to make this happen. Neither properties can depend on each other like this, nor there is support for button groups (otherwise there could be a `ZoomInOutTool`).